### PR TITLE
Next

### PR DESCRIPTION
**Pull request by G-Rath**
This is a body for my PR

### CODE REVIEW

File: package.json
Comment (G-Rath) [inline package.json:28]: @SimenB  `src/globals.json` 😬

**Comment by SimenB** [comment]
@G-Rath do you wanna hold off for v4 of `@typescript/eslint`? Either way works for me. I doubt the changes in the utils will affect us, but who knows

**Comment by G-Rath** [comment]
@SimenB I was reading some stuff about it today in relation to prettier, and it sounded like it was [very close to being released](https://github.com/prettier/prettier/issues/8882#issuecomment-669017823).

v4 has some shiny new packages like `@typescript-eslint/scope` that *might* be useful for us, and that I suspect won't play nice with the v3 components if we tried to use them, so unless theres a specific breaking change people are waiting on, I don't think theres much harm in waiting at least another week to see where `@typescript-eslint` is at.

(iirc we're pretty stable these days, so this major is just letting us internally bump a bunch of dependencies by dropping Node 8, but I could be forgetting something?)

**Comment by SimenB** [comment]
There are no changes in the util we use I believe, unless I misunderstand. Regardless, there is not rush in landing this. `typescript@4` is supposed to come next week, that'll probably kick off some releases: https://github.com/microsoft/TypeScript/issues/38510

**Comment by G-Rath** [comment]
@SimenB do we want to wait another week? I'm pretty busy with work stuff this week myself, so it fits with my schedule on getting the last few things ready to go.

https://github.com/typescript-eslint/typescript-eslint/pull/2330#issuecomment-678871683

**Comment by SimenB** [comment]
yeh, no rush

**Comment by SimenB** [comment]
Updated to ts-eslint@3.10, but we'll hold off until next week to update to v4

**Comment by SimenB** [comment]
@G-Rath v4 made no difference to types or rules. Should we merge?

**Comment by G-Rath** [comment]
@SimenB I'm keen to merge, but would like to do so final checks - would you be ok waiting until tonight/tomorrow (iirc my tonight is your ~3am)?

I'm working in a makeshift space today as we've not got power today at home, so working o-natural with just the built-in laptop 
peripherals :joy:

**Comment by SimenB** [comment]
For sure, still no rush 🙂 Make sure to land #651 before (rebase) merging this one 👍

**Review by SimenB — APPROVED** [review]
Let's do it

**Comment by SimenB** [comment]
Woo! Thanks @G-Rath, this was a heavy one 🚀

**Comment by G-Rath** [comment]
@SimenB we didn't remove our deprecated rules 😬 

This [branch](https://github.com/jest-community/eslint-plugin-jest/tree/remove-rules) never got a PR opened.

**Comment by SimenB** [comment]
Bah. Next time 😃 Open a PR with it now so we don't lose track?

**Pull request merged**
